luno.go

```diff
@@ -151,7 +150,7 @@ func (cl *Client) do(ctx context.Context, method, path string,
 
 	body = httpRes.Body
 	if cl.debug {
-		b, err := ioutil.ReadAll(body)
+		b, err := io.ReadAll(body)
```

Comment:
staticcheck flag:
```shell
luno.go:11:2: "io/ioutil" has been deprecated since Go 1.19: As of Go 1.16, the same functionality is now provided by package [io] or package [os], and those implementations should be preferred in new code. See the specific function documentation for details.  (SA1019)
```